### PR TITLE
Feature/game display

### PR DESCRIPTION
**Pull request by JJoeeyy**
### What?
Start of Frontend-View

### Why?
To have a static Frontend-View, on which we can start to build

### Screenshots (for frontend changes)
TODO

### CODE REVIEW

**Comment by Oliver-was-taken** [comment]
Please remove .bin from pr and add it to .gitignore

**Review by Oliver-was-taken — CHANGES_REQUESTED** [review]
Remove node_modules from pr

**Comment by keykey7** [comment]
you are allowed to modify (or even drop) `prettifier`, but you have to pass the npm test :P

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "sass": "^1.67.0"
+  }
```
Comment (corsin-lehre) [inline package.json:4]: This doesn't look quite right... Can you revert this change?

**Review by corsin-lehre — CHANGES_REQUESTED** [review]

**Review by corsin-lehre — APPROVED** [review]
LGMT

**Review by Desi7 — APPROVED** [review]
Funny thinking to use an eraser, because of the paper 👍 code as well good

**Review by Lion-alt — APPROVED** [review]
Looks good 👍

File: fabbwled-frontend/src/routes/game/Game.tsx [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+          </p>
+        </div>
+        <div className={styles.page}>
+          <div className={styles.statsDisplay}>
+            <h3>Xisaj Nikolaj</h3>
+            <div className={styles.stats}>
+              {/* TODO: Replace this with table */}
+              <ul>
+                <li>Rank: 1</li>
+                <li>Profession: Hurensohn</li>
```
Comment (MagischerWolf05) [inline fabbwled-frontend/src/routes/game/Game.tsx]: wie bitte

**Review by Desi7 — APPROVED** [review]
Nice 👍

**Review by MagischerWolf05 — APPROVED** [review]
TOp notch

**Pull request merged**